Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/task.json

```diff
@@ -215,7 +215,7 @@
             "Expression": "eq(jsonpath('$.result[0].status')[0], 'inserted')"
           },
           "ExecutionOptions": {
-            "OutputVariables": "{\"CHANGE_SYSTEM_ID\" :  \"jsonpath('$.result[0].sys_id')[0]\", \"CHANGE_REQUEST_NUMBER\" :  \"jsonpath('$.result[0].display_value')[0]\"}",
```

Comment:
This will be a breaking change as Change_Request_Number might not be set in some flows.